x-pack/platform/plugins/shared/onechat/public/application/components/conversations/conversation_input/conversation_input_text_area.tsx

```diff
@@ -51,6 +55,18 @@ export const ConversationInputTextArea: React.FC<ConversationInputTextAreaProps>
       textAreaRef.current?.focus();
     }, 200);
   }, [conversationId]);
+
+  const disabledPlaceholder = i18n.translate(
+    'xpack.onechat.conversationInputForm.disabledPlaceholder',
+    {
+      defaultMessage:
+        'The following agent could not be found: {agentId}. Please start a new conversation.',
```

Comment:
```suggestion
        "Agent '{agentId}' has been deleted. Please start a new conversation.",
```

Reply:
https://github.com/elastic/kibana/pull/236410/commits/7b9b55093e343d51162e0a61996c7e202504eb3e does this work? Had to swap around the quotes to get valid syntax

Reply:
LGTM!